### ISSUE TITLE
Import of multiple columns with time, concentration, and error

### ISSUE THREAD

**Issue by UlrichSi** [Open-Systems-Pharmacology/PK-Sim#857]
The attached file contains PK data with time, concentration, and SD. When importing it, each column appears to be combined with the other columns in multiple ways. Seems to be related to [MoBi issue #193](https://github.com/Open-Systems-Pharmacology/MoBi/issues/193). 

In a similar way, a table containing only one time column and several column pairs of concentration/error will produce too many useless import columns.

If there is a scenario where the current import behavior is desired, I would suggest to have it optionally switched off, since our current data scanning software produces such attached files.

[Mappe1.xlsx](https://github.com/Open-Systems-Pharmacology/PK-Sim/files/2119182/Mappe1.xlsx)

**Comment by msevestre** [comment]
I am not sure why this is implemented like that. I cannot think of a scenario where the full combinatoric would make sense.

The only use case where that makes sense is really one time column and N concentration columns. In that case, create N data set. Fine

At that stage however, we will not fix anything in the Importer. The code is simply impossible to work with and every single change can lead to major unexpected issues. 

I'd like to start gathering all sorts of improvement that could be done in the importer (both UI and functionalities) as we are going to reimplement it from scratch this year

**Comment by msevestre** [comment]
Task: Remove combinatoric nonsense

**Comment by UlrichSi** [comment]
> The only use case where that makes sense is really one time column and N concentration columns. In that case, create N data set. Fine

If there are only one time column and N alternating columns of concentration and error, that will also cause a combinatorial disaster.
I guess the implementation was just created by simply using this use case you mentioned, but not thinking of having additional error columns or multiple time columns.

**Comment by msevestre** [comment]
yeah. To be honest, I am not sure how often we actually have this use case in real scenario

**Comment by UlrichSi** [comment]
If the code will be redesigned, I would recommend to made compatible with the latest versions of common data scanning software tools used by the people out there.

**Comment by msevestre** [comment]
I'd suggest that you start a list of what those tools are :). We have a fix tight budget so the list should focus on what REALLY matters. I'd also suggest to compile that list in another issue
